featuretools/primitives/__init__.py

```diff
@@ -5,41 +5,47 @@
 
 from .api import *  # noqa: F403
 
-# Load in a list of primitives registered by other libraries into Featuretools.
-#
-# Example entry_points definition for a library using this entry point:
-#
-#    entry_points={
-#        "featuretools_primitives": [
-#            other_library = other_library:LIST_OF_PRIMITIVES
-#        ]
-#    }
-#
-# where `LIST_OF_PRIMITIVES` is an iterable of primitive class objects defined
-# in module `other_library`.
-
-for entry_point in pkg_resources.iter_entry_points('featuretools_primitives'):  # pragma: no cover
-    try:
-        loaded = entry_point.load()
-    except Exception as e:
-        logging.warning(
-            "entry point \"%s\" in package \"%s\" threw exception while loading: %s",
-            entry_point.name,
-            entry_point.dist.project_name,
-            repr(e),
-        )
-        continue
-
-    for primitive in loaded:
-        if primitive.__name__ in globals():
-            raise RuntimeError(
-                f"primitive with name \"{primitive.__name__}\" already exists"
+
+def _load_primitives():
+    """Load in a list of primitives registered by other libraries into Featuretools.
+
+        Example entry_points definition for a library using this entry point:
+        entry_points={
+            "featuretools_primitives": [
+                other_library = other_library:LIST_OF_PRIMITIVES
+            ]
+        }
```

Comment:
```suggestion
        If using setup.py:
        
        entry_points={
            "featuretools_primitives": [
                other_library = other_library.file_with_primitives_list:LIST_OF_PRIMITIVES
            ]
        }
        
        If using setup.cfg:
        
            [options.entry_points]
            featuretools_primitives =
                other_library = other_library.file_with_primitives_list:LIST_OF_PRIMITIVES
```

Reply:
@jeff-hernandez Should the other_library have the primitive list in the `__init__.py` or a separate file? I feel like having it in the `__init__.py` might lead to a circular import

Reply:
Pointing to the `__init__.py` isn't necessary, just the module that contains the primitive objects. Ideally, the other primitive libraries will have access to all the primitives from the top-level module.

Reply:
But shouldn't be put an example here + note that says not to use `__int__.py` in your other_library? To avoid a circular import? Because if you try to import you other_library, it might have to import `featuretools` which has to import the other library.

Reply:
I think `other_library = other_library` and `other_library = other_library.__init__.py` are equivalent, so if a circular import raises in one then it should raise in the other. I haven't tested whether or not an `import featuretools` statement inside that module would raise a circular error, but it's worth testing.

Reply:
With this entrypoint, you just have to point to the file where all primitives are imported (which could be the __init__.py` of the library).